android/app/src/main/AndroidManifest.xml

```diff
@@ -51,6 +51,19 @@
         </intent-filter>
       </activity>
       <activity android:name="com.facebook.react.devsupport.DevSettingsActivity" />
+        <activity
+            android:noHistory="true"
+            android:name=".share.ShareActivity"
+            android:configChanges="orientation"
+            android:label="@string/share_extension_name"
+            android:screenOrientation="portrait"
+            android:theme="@style/AppTheme" >
+            <intent-filter>
+                <action android:name="android.intent.action.SEND" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <data android:mimeType="text/plain" />
```

Comment:
Maybe should use
```
<data android:scheme="http" />
<data android:scheme="https" />
```
?

Reply:
Should use MIME type, https://developer.android.com/training/sharing/send#using-the-right-mimetype

Reply:
Can I add `android:scheme` together with mime type to ensure the string passed is really a url?

Reply:
Liker Land disappeared after adding scheme

For Pocket, it can share plain text with corresponding error message, we may follow its behavior
![image](https://user-images.githubusercontent.com/17264716/77031474-ae869a80-69dc-11ea-8b7e-f1c50605ba61.jpeg)

Reply:
Ok lets handle with app logic then